frontend/src/lib/i18n/en.governance.json

```
@@ -19,8 +19,8 @@
     "Kyc": "KYC",
     "NodeProviderRewards": "Node Provider Rewards",
     "SnsDecentralizationSale": "SNS Decentralization Swap",
-    "ReplicaVersionManagement": "Replica Version Management",
-    "SubnetReplicaVersionManagement": "Subnet Replica Version Management",
+    "ReplicaVersionManagement": "IC OS Version Election",
```

Shouldn't we also rename the internal names for consistency?

```suggestion
    "IcOsVersionElection": "IC OS Version Election",
```

---

It would also be a breaking change for any other project using that library so I'm not sure we should even do it.

---

Regardless, the TODO would belong in the other repo. Not here.

---

I have added a comment in the ticket to consider it:
https://dfinity.atlassian.net/browse/NNS1-3003